docs/orm-integration.md

```diff
@@ -193,21 +209,17 @@ Then modify your strategy's \_\_init__ function to match the following code:
             self._data_for_sale = self.collect_from_data_source()
         else:
             self._data_for_sale = kwargs.pop("data_for_sale", DEFAULT_DATA_FOR_SALE)
-
-        super().__init__(**kwargs)
```

Comment:
this was too late... `self.create_database_and_table()` needs the `self.context` to be set, which is done in the parent constructor.

Reply:
Note, by moving it above the `kwargs.pop`  you still end up passing all the kwargs to super, which will then cause a logger warning. How about keeping it here and instead moving 
```
self._tbl = self.create_database_and_table()
self.insert_data()
```
down?